src/server/game/DataStores/DB2Stores.h

```diff
@@ -276,6 +280,7 @@ class TC_GAME_API DB2Manager
     typedef std::tuple<int16, int8, int32> WMOAreaTableKey;
     typedef std::map<WMOAreaTableKey, WMOAreaTableEntry const*> WMOAreaTableLookupContainer;
     typedef std::unordered_map<uint32, WorldMapAreaEntry const*> WorldMapAreaByAreaIDContainer;
+    typedef std::tuple<uint32, uint32> PvpRewardKey;
```

Comment:
you could have used std::pair for this and just went with regular unordered_map - we have a pair hasher in Common.h